lib/types/index.ts

```diff
@@ -93,6 +93,7 @@ export type {
   EntrySys,
   ContentEntitySys,
   ContentEntityType,
+  FieldType,
```

Comment:
Also export `FieldLinkType`?

Reply:
Do you think we should expose it?

Reply:
We export types (`EntryFieldAPI`) that use this type, so yes.

Reply:
Done